.changeset/wild-sheep-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Enhanced userPresence options making it more human readable.
```

Comment:
Since this is customer facing you could add more details

```
Added input fields so the user can specify exactly the time of the enableAutoAway preference in hours, minutes and seconds.
```